validation/cross_validate_rust_python.py

```
@@ -54,12 +54,28 @@
+
+    print(f"  R_c = {R_c_rust*1e6:.4f} μm")
+    print(f"  P_Blake = {P_Blake_rust:.2f} Pa = {P_Blake_rust/1000:.2f} kPa")
+
+    rc_diff = abs(R_c_python - R_c_rust) / R_c_python * 100
+    pb_diff = abs(P_Blake_python - P_Blake_rust) / P_Blake_python * 100
+
+    print(f"  R_c difference: {rc_diff:.6f}%")
+    print(f"  P_Blake difference: {pb_diff:.6f}%")
+
+    if rc_diff < 1e-4 and pb_diff < 1e-4:
+        print(f"  ✓ Cross-validation passed")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)